Easydict/NewApp/View/SettingView/Tabs/DisabledAppTab.swift

```diff
@@ -118,11 +115,11 @@ struct DisabledAppTab: View {
     var body: some View {
         VStack {
             Text("disabled_title")
-                .padding()
+                .padding(.top, 18)
+                .padding(.bottom, 8)
```

Comment:
This will change text horizontal padding. 

<img width="496" alt="Screenshot 2024-01-20 at 08 43 43" src="https://github.com/tisfeng/Easydict/assets/103433299/014dce3e-44e2-4c0f-adf0-8496d2073a18">

Reply:
ok